parlai/mturk/core/scripts/bonus_workers.py

```diff
@@ -7,6 +7,11 @@
 import argparse
 import parlai.mturk.core.mturk_utils as mturk_utils
 
+print(
+    'This script will be removed from this location May 1st 2019. '
```

Comment:
lol maybe refactor out your deprecation warning?

Reply:
I'm not sure there are enough cases of this to be worth refactoring out.